docs/develop/python/testing-suite.mdx

```diff
@@ -36,6 +36,30 @@ Some SDKs have support or examples for popular test frameworks, runners, or libr
 
 One recommended framework for testing in Python for the Temporal SDK is [pytest](https://docs.pytest.org/), which can help with fixtures to stand up and tear down test environments, provide useful test discovery, and make it easy to write parameterized tests.
 
+### A note on pytest fixtures
+
+When using pytest fixtures to setup databases, application servers and common infrastructure it is not unusual to use the `session` scope to ensure the fixture is shared among multiple tests. However, with the time-skipping test server it is important that each test is executed in isolation. Launching a test server per test introduce negligible overhead and ensures your test do not conflict with each other
+
+
+```python
+import pytest
+import temporalio.testing.WorkflowEnvironment
+
+
+# Notice how this doesn't specify a "scope" like "session" scope. If you don't
+# specify a scope, the default scope is function, so a new workflow
+# environment will be created for each test.
+@pytest.fixture
+def workflow_environment():
+  return WorkflowEnvironment.start_time_skipping()
```

Comment:
I am not sure this works without `@pytest_asyncio.fixture` and `async` and `await`. Have you tested this code? And for many, they may prefer a "session" scoped `start_local` instead of time skipping even though this only demonstrates time skipping.

Reply:
That's a good point, was using the auto-mode in my tests https://pytest-asyncio.readthedocs.io/en/latest/reference/configuration.html  but I agree that being explicit works better